docs/interpreting-experiments/cuped.md

```diff
@@ -1,15 +1,20 @@
 # Controlled-experiment Using Pre-Existing Data (CUPED)
-CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. If data is music, CUPED is a pair of noise canceling headphones, allowing you to notice a more pronounced pattern in the data despite a lower volume or strong external noise.
+The precision of experiment results (the width of confidence intervals) depends on the variance in of the metrics we are measuring. One way to improve the precision of experiments is to gather more data, as the variance goes down as we gather more data; this obviously means that it takes longer to run an experiment. 
+
+However, there are options beyond waiting to gather more data, and all these options address reducing the variance in the metrics we are measuring directly. One particular flexible and powerful method is known as CUPED. CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. You can think of CUPED as a pair of noise cancelling headphones: it uses data gathered prior to the experiment to understand the ambient noise, allowing you to notice a more pronounced pattern in the data.
```

Comment:
I think the first sentence is confusing. How about "However, there are more direct ways to reduce variance, beyond just waiting for more data to show up."

Also, "One particularLY flexible and powerful method"? I would also spell out the acronym: "known as CUPED, which stands for Controlled...[etc]."

I would also change "pre-existing": what's relevant is that it is data from _before_, or more properly _outside_ the experiment. How about something like "Standard experiment analysis involves comparing metric data from subjects exposed to a treatment to that of a control group -- all the data is gathered _during_ the experiment. But most companies know something about their users beyond what they did during the experiment, most prominently metric data from _before_ the experiment started. CUPED leverages this data from outside the experiment in order to control for some of the variance in metrics that comes from randomly picking variants for each subject; in a standard experiment, you might end up with one variant having more active users just by random chance, while CUPED reduces the effect of this random variation by _controlling for_ the different activity levels across different variants."